src/shared/constants/fluxFunctions.ts

```diff
@@ -1,5 +1,7 @@
 import {FluxToolbarFunction} from 'src/types/shared'
 
+export const DOCS_URL_VERSION = !!process.env.CLOUD_URL ? 'cloud' : 'v2.0'
```

Comment:
is it possible to fetch this version from somewhere? docs will go to v2.1 and we have to remember to change this value? seems tricky.

Reply:
i think you can just use `latest` instead of `v2.0` and it would be fine.

Reply:
Makes sense to use `latest` - especially on the master UI branch. We might have to think about how we want this to work on release branches eventually, otherwise a 2.1 release will link to the 2.2 docs etc. if it just links to latest. Maybe that's alright though.

Reply:
This is updated now - will use the latest URL on OSS.